### DISCUSSION TITLE
Newby here. What are the different output folders for.

### DISCUSSION THREAD

**Discussion by jrodh13** [automatic-ripping-machine/automatic-ripping-machine#1134] [Q&A]
I want to understand what the three output folders are for. I've got raw, trans code, and completed (as I'm sure your all aware). I tend to see the same files in both the raw and transcode folders and they are both playable with VLC. 

What are differences here? Could someone please help me understand?

**Comment by SylvainMT** [comment]
- Raw: The files outputted by MakeMKV (the first phase of the ARM process)
- Transcode: The files outputted by HandBrake (These are the compressed files using the HB preset you chose, created by reading the raw files, the second step in the ARM process)
- Completed: Once all the transcoding is done, the last phase of the ARM process is to copy the files to the completed folder and then delete the files in the transcode and raw folders (The last step in the ARM process)

**Reply by jrodh13** [reply]
Ah!!! Ok! I didn't realize the other files would be deleted. Thank you for the explanation. 

You mentioned "the HB preset you chose". I'm glad you mentioned that because it led me the ripper settings page where I saw the option HB_PRESET_DVD. Hovering over the info icon associated with that setting gave me the command in the terminal to see the available options. I had another question I posted after this one where I asked if I could convert the file to MP4, which it appears to already be set to, based on the HB settings returned from the terminal. I am assuming then that the file/s in the completed window should be in MP4 format?

**Reply by SylvainMT** [reply]
So yes, HandBrake can output MP4 format.  This is dependent on the preset you choose.  Unfortunately, the info box in the ARM UI is still operating on the fact that ARM used to be primarily installed bare-metal, now that it is installed as a Docker container you need to run this command;

`docker exec -t arm-rippers HandBrakeCLI -z`

That would be the correct command to run now.  This assumes that you have not changed the --name parameter inside the `start_arm_container.sh` script.

You can also create your own preset by creating one inside of a Desktop installation of HandBrake (Linux or Windows) exporting that preset file and then importing it in arm.

You can do so by filling in the field "HB_ARGS_DVD: " (or "HB_ARGS_BD: " for Blu Rays) with the value:
`--preset-import-file <Path-to-preset-file-accessible-by-the-arm-user>`
